src/relayServerUtils.ts

```diff
@@ -35,43 +39,99 @@ async function calculateFee(
   relayRequest: EnvelopingRequest,
   maxPossibleGas: BigNumber,
   appConfig: AppConfig
-) {
+): Promise<BigNumberJs> {
+  if (isSponsorshipAllowed(relayRequest, appConfig)) {
+    return BigNumberJs(0);
+  }
+
+  const { fixedUsdFee } = appConfig;
+  let totalFee = BigNumberJs(0);
```

Comment:
I'd rename this variable
```suggestion
  let fixedUsdFee = BigNumberJs(0);
```

Reply:
Using fixedFee here because fixedUsdFee was already here `const { fixedUsdFee } = appConfig;`